### PR TITLE
Userguide and addNote bugs

### PR DESCRIPTION
**Pull request by Nazryl**
- [x] #154 
- [x] #167
- [x] #172
- [x] #155

### CODE REVIEW

**Comment by R-Ramana** [comment]
> * [x]  #154
> * [x]  #167
> * [x]  #172

If your intention was to close the issue this doesn't do it. You need to use the following format:
Closes #num1
Closes #num2

For more examples and usage of other keywords refer to the following [document](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).

File: docs/UserGuide.md [hunk 1]
```diff
@@ -102,8 +102,8 @@ Format: `add-n /t TITLE [/tag TAG_1] [/tag TAG_2]... [/pin ISPIN] [/archive ISAR
 
 💡 Each note has to have a **UNIQUE** title and it is **CASE-INSENSITIVE**. <br>
 💡 One can choose to add a `TAG` or/and `ISPIN`, `ISARCHIVE`. These parameters are optional.<br>
-💡 Set `ISPIN` to true if you want the note to be pinned.  
-💡 Set `ISARHCIVE` to true if you want the note to be archived.
+💡 Set `ISPIN` to "true" if you want the note to be pinned and any other value for an unpinned note.  
+💡 Set `ISARCHIVE` to "true" if you want the note to be archived and any other value for an unarchived note.
```
Comment (R-Ramana) [inline docs/UserGuide.md:106]:
```suggestion
💡 Set `ISPIN` to "true" if you want the note to be pinned. Any other input value will leave the note unpinned.  
💡 Set `ISARCHIVE` to "true" if you want the note to be archived. Any other input value will leave the note unarchived.
```

File: src/main/java/seedu/notus/util/CommandMessage.java [hunk 2]
```diff
@@ -232,7 +232,9 @@
     public static final String UNARCHIVE_NOTE_SUCCESSFUL_MESSAGE = colorize("The following note has been unarchived: ",
             GREEN);
 
-    public static final String NOTE_UNSUCCESSFUL_MESSAGE = colorize("This note already exists in the notebook! ", RED);
+    public static final String NOTE_EXIST_MESSAGE = colorize("This note already exists in the notebook! ", RED);
+    public static final String NOTE_ARCHIVED_EXIST_MESSAGE =
+            colorize("This note already archived in the notebook! ", RED);
```
Comment (R-Ramana) [inline src/main/java/seedu/notus/util/CommandMessage.java:237]:
```suggestion
            colorize("This note can be found in the archived list! ", RED);
```

**Review by R-Ramana — APPROVED** [review]

**Comment by Nazryl** [comment]
> > * [x]   #154
> > * [x]   #167
> > * [x]   #172
> 
> If your intention was to close the issue this doesn't do it. You need to use the following format:
> Closes #num1
> Closes #num2
> 
> For more examples and usage of other keywords refer to the following [document](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).

Oh I know. it just serves me as a checklist

**Comment by Nazryl** [comment]
Thanks bru! @R-Ramana

**Comment by Nazryl** [comment]
Closes #154 
Closes #167
Closes #172
Closes #155

**Pull request merged**